docs/utils/use-atom-value.mdx

```diff
@@ -1,5 +1,6 @@
 ---
 title: useAtomValue
+status: deprecated
```

Comment:
`useAtomsValue` and `useUpdateAtom` from `jotai/utils` are not really deprecated in code *yet*.

Reply:
@dai-shi Sorry, but what do you mean by "yet"?

You want to "force" usage of useAtom ?
`const [val] = useAtom(...)`

I liked the useAtomValue pattern a lot?

On the other hand, why not state `useUpdateAtom` as deprecated now? As it's [already mentioned in the documentation](https://jotai.org/docs/utils/use-update-atom) to use `useSetAtom` instead, for newer versions.